src/Common/tests/TestUtilities/System/AssertExtensions.cs

```diff
@@ -191,867 +191,5 @@ public static Exception Throws(Type netCoreExceptionType, Type netFxExceptionTyp
                 Throws<TNetCoreExceptionType>(netCoreParamName, action);
             }
         }
-
```

Comment:
Why were these lines removed?

Also, isn't AssertExtensions available via a nuget package that can be added as a dependency?

Reply:
The better question to ask, why were those lines added in the first place, when not needed... We want to only check the code in that our libraries actually need.

No, AssertExtensions.cs is checked into runtime directly.